nwb_healthstatus/producers/spikeextractors/recording.py

```diff
@@ -0,0 +1,278 @@
+            "kwik",
+            dict(file_or_folder_path=Path("kwik", "neo.kwik")),
+        ),
+        # Neuralynx - units_ids = nwbfile.units.id[:] - AttributeError: 'NoneType' object has no attribute 'id'
+        # Is the GIN data OK? Or are there no units?
+        # (
+        #     se.NeuralynxSortingExtractor,
+        #     "neuralynx/Cheetah_v5.7.4/original_data",
+        #     dict(
+        #         dirname=Path("neuralynx", "Cheetah_v5.7.4", "original_data"),
+        #         seg_index=0
+        #     )
+        # ),
+        # NIXIO - return [int(da.label) for da in self._spike_das]
+        # TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
+        # (
+        #     se.NIXIOSortingExtractor,
+        #     "nix/nixio_fr.nix",
+        #     dict(file_path=Path("nix", "nixio_fr.nix")),
+        # ),
+        (
+            se.MEArecSortingExtractor,
+            "mearec/mearec_test_10s.h5",
+            dict(file_path=Path("mearec", "mearec_test_10s.h5")),
+        ),
+        (
+            se.PhySortingExtractor,
+            "phy/phy_example_0",
+            dict(folder_path=Path("phy", "phy_example_0")),
+        ),
+        (
+            se.PlexonSortingExtractor,
+            "plexon",
+            dict(filename=Path("plexon", "File_plexon_2.plx")),
+        ),
+        (
+            se.SpykingCircusSortingExtractor,
+            "spykingcircus/spykingcircus_example0",
+            dict(
+                file_or_folder_path=Path(
+                    "spykingcircus", "spykingcircus_example0", "recording"
+                ),
+            ),
+        ),
+        # # Tridesclous - dataio error, GIN data is not correct?
+        # (
+        #     se.TridesclousSortingExtractor,
+        #     "tridesclous/tdc_example0",
+        #     dict(folder_path=Path("tridesclous", "tdc_example0")),
+        # )
+    ]
+
+    def create(self):
+        for se_class, dataset_path, se_kwargs in self.SCENARIOS:
+            self.dataset.get(dataset_path)
+            for k in [
+                "filename",
+                "file_or_folder_path",
+                "file_path",
+                "dirname",
+                "folder_path",
+            ]:
+                if k in se_kwargs:
+                    se_kwargs[k] = self.pt / se_kwargs[k]
+            if "filename" in se_kwargs:
+                se_kwargs["filename"] = str(se_kwargs["filename"])
+            sorting = se_class(**se_kwargs)
+            sf = sorting.get_sampling_frequency()
+            if (
+                sf is None
+            ):  # need to set dummy sampling frequency since no associated acquisition in file
+                sf = 30000
+                sorting.set_sampling_frequency(sf)
+            with tempfile.NamedTemporaryFile() as tf:
+                se.NwbSortingExtractor.write_sorting(sorting, tf.name)
+                with pynwb.NWBHDF5IO(tf.name, mode="r") as io:
+                    nwbfile = io.read()
```

Comment:
@yarikoptic I got the Extractors to run, but the Sorters are failing here with `hdmf.backends.io.UnsupportedOperation: Cannot read data from file /var/folders/l7/wrkq93d133d8zpn36fmqrq0r0000gn/T/tmpz8otn54x in mode 'r+'. There are no values.`.

Reply:
is it only for a single extractor or for all of them?

Reply:
if single -- just comment it out for now and produce for those for which it produces, and report for those which do not.
If for all -- I guess it is something more fundamental - just add a comment and do not enable the sorters tests

Reply:
@yarikoptic It's for all of them.